{{cookiecutter.project_slug}}/backend/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build all of the dependencies then we will build the actual application image
-FROM python:3.12-slim as build
+FROM localhost:5001/python:3.12-slim as build
```

Comment:
Please don't hardcode the local registry in the Dockerfile. This leaves us with a Dockerfile that is not portable to different environment and it won't build in CI.